.env

```diff
@@ -0,0 +1,7 @@
+# Twilio API credentials
+export TWILIO_ACCOUNT_SID=AC807f0beda34c3b254e0cf785a42420af
+export TWILIO_AUTH_TOKEN=4de3c67a78763ae5c4e67a2cadce45f9
+
+# Twilio sending number
+export TWILIO_NUMBER=+13143000470
+
```

Comment:
This too is private information could be stored directly on Heroku as variables defined
Tokens should always remain private, reason why we place them in the environment 
all of the variables can be stored on Heroku variables where the Database URL variable is defined

You should send a sample.env not the .env file :-)

Reply:
Noted. Thanks for the heads up